src/test/java/seedu/linkedout/model/applicant/JobTest.java

```diff
@@ -51,34 +51,21 @@ public void isValidJob_whitespace_false() {
 
     @Test
     public void isValidJob_nonAlphaNumeric_false() {
+        // Partition of symbols with ascii values 33-47
+        assertFalse(Job.isValidJob(String.valueOf((char) ASCII_CHARACTER_EXCLAMATION_MARK)));
```

Comment:
correct me if I'm wrong but this seems like it is only testing for ascii value 33 and 47 instead of 33-47?

Reply:
Same for other changes